AWS GuardDuty support authentication via tokens support (#15235)
demisto/content: Packs/AWS-GuardDuty/ReleaseNotes/1_2_0.md

#### Integrations
##### AWS - GuardDuty

> - Added the *Access Key* and *Secret Key* parameters which enable you to authenticate in another way.

Reviewer: ```suggestion
- Added the *Access Key* and *Secret Key* parameters to enable authentication using without a *roleArn*.
```

> - Added the *Timeout* parameter, which enables you to define the amount of time until a timeout error occurs.

Reviewer: ```suggestion
- Added the *Timeout* parameter to enable configuration of the amount of time until a timeout error occurs.
```

> - Added the *Retries* parameter, which enables you to define the maximum retry attempts until a connection or throttling error occurs.

Reviewer: ```suggestion
- Added the *Retries* parameter, to enable configuration of the maximum retry attempts until a connection or throttling error occurs.
```
